### DISCUSSION TITLE
Local DNS Records

### DISCUSSION THREAD

**Discussion by UnNamed-M3** [AdguardTeam/AdGuardHome#3728] [Q&A] [answered]
Hi,

I was wondering how I could configure AdGuard Home to create my own local A and CNAME DNS Records? 

Many thanks.

Best Regards,

**Comment by ainar-g** [comment, marked as answer]
Hello.  You can do that either through the older mechanism of “DNS rewrites” (under the “Filtering” menu in the UI) or using the newer and more flexible [`$dnsrewrite` custom rules][1].

[1]: https://github.com/AdguardTeam/AdGuardHome/wiki/Hosts-Blocklists#dnsrewrite

**Reply by UnNamed-M3** [reply]
Many thanks ainar-g!! Somehow I totally missed that option in the UI! For now this exactly does what I want it to do!
